src/main/java/uk/gov/hmcts/et/common/model/ccd/CaseData.java

```diff
@@ -441,4 +441,12 @@ public class CaseData extends Et1CaseData {
     private String et3RegionalOffice;
     @JsonProperty("et3WhyWeShouldChangeTheOffice")
     private String et3WhyWeShouldChangeTheOffice;
+
+    // ET Initial Consideration
+    @JsonProperty("etInitialConsiderationRespondent")
+    private String etInitialConsiderationRespondent;
+    @JsonProperty("etInitialConsiderationHearing")
+    private String etInitialConsiderationHearing;
+    @JsonProperty("etInitialConsiderationJurisdictionCodes")
+    private String etInitialConsiderationJurisdictionCodes;
```

Comment:
Looking at the rest of the properties we either prefix the et number (ie "et3") or we don't prefix anything to the fields. It would be better to either remove the "et" prefix or add the number onto it